joda/src/main/scala/org/scalacheck/ops/time/joda/package.scala

```diff
@@ -1,3 +1,10 @@
 package org.scalacheck.ops.time
 
-package object joda extends JodaTimeImplicits
+package object joda extends JodaTimeImplicits {
+
+  @deprecated("Use JodaDateTimeGenerators instead.", "2.0.0")
+  type DateTimeGenerators = JodaDateTimeGenerators
+
+  @deprecated("Use JodaLocalDateTimeGenerators instead.", "2.0.0")
+  type LocalDateTimeGenerators = JodaLocalDateTimeGenerators
```

Comment:
I renamed the Joda stuff, and there are a lot of binary breaking changes, but I don't want to introduce source code incompatibilities if I don't have to. I want people's imports to work as normal.